scite/parser.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+
+'use strict';
+const Parser = require('../.lib/parser.js');
+
+/**
+ * Recognizes the accesses to the platform Scite
+ * @param  {Object} parsedUrl an object representing the URL to analyze
+ *                            main attributes: pathname, query, hostname
+ * @param  {Object} ec        an object representing the EC whose URL is being analyzed
+ * @return {Object} the result
+ */
+module.exports = new Parser(function analyseEC(parsedUrl, ec) {
+  let result = {};
+  let path   = parsedUrl.pathname;
+  // uncomment this line if you need parameters
+  let param = parsedUrl.query || {};
+
+  // use console.error for debuging
+  // console.error(parsedUrl);
+
+  let match;
+
+  if (/^\/search$/i.test(path) && (param.q !== undefined || param.author !== undefined)) {
+    // https://scite.ai/search?mode=all&q=Using%20AI%20to%20Teach%20AI
+    // https://scite.ai/search?mode=all&q=Keeping%20it%20real
+    // https://scite.ai/search?mode=all&author=David%20Smith
+    // https://scite.ai/search?mode=all&author=John%20Edwards
+    result.rtype    = 'SEARCH';
+    result.mime     = 'HTML';
+    if (param.q) {
+      result.title_id = param.q;
```

Comment:
It Is not necessary to add "unitid" and "title_id" on SEARCH

Reply:
Hi,

I have removed these fields.
Please check. Thanks